templates/output.html.twig

```diff
@@ -31,8 +31,13 @@
                 </div>
             </div>
             <div class="form-group">
-                <label for="lang">{{ msg('language-code') }}</label>
-                <select id="lang" class="form-control" name="langs[]" multiple data-placeholder="{{ msg('langs-placeholder') }}">
+                <label id="optional-lang-label" for="lang" class="lang-label {% if engine == 'transkribus' %}hidden{% endif %}">
+                    {{ msg('language-code') }}
+                </label>
+                <label id="transkribus-lang-label" for="lang" class="lang-label {% if engine != 'transkribus' %}hidden{% endif %}">
+                    {{ msg('transkribus-language-code') }}
+                </label>
+                <select id="lang" class="form-control" name="langs[]" multiple data-placeholder="{{ msg('langs-placeholder') }}" {% if engine == 'transkribus' %}required{% endif %}>
```

Comment:
Sorry, I've just realised that `langs-placeholder` is "Leave blank for automatic language detection." which isn't correct if it's a required field. Probably it can just be made blank for Transkribus, rather than adding a new message to be translated (but do whatever you feel is best).